src/net_processing.cpp

```diff
@@ -4335,7 +4337,7 @@ void PeerManagerImpl::CheckForStaleTipAndEvictPeers()
 
     int64_t time_in_seconds = GetTime();
```

Comment:
```suggestion
    auto time_in_seconds = GetTime<std::chrono::seconds>();
    
    EvictExtraOutboundPeers(time_in_seconds);
```

Reply:
The diff as suggested wouldn't compile and needs more changes, so I'll leave it for a follow-up.